biogtr/models/embedding.py

```diff
@@ -13,12 +13,117 @@ class Embedding(torch.nn.Module):
     Used for both learned and fixed embeddings.
     """
 
-    def __init__(self):
-        """Initialize embeddings."""
+    EMB_TYPES = {
+        "temp": {},
+        "pos": {"over_boxes"},
+        "": {},
```

Comment:
call this `"null"` or `"zeros"` or something that's more descriptive than an empty string so the behavior is a bit clearer from the arguments.